.github/workflows/pr_push.yml

```diff
@@ -14,50 +14,84 @@ concurrency:
 
 permissions:
   contents: read
+  packages: write
```

Comment:
pls only add this extra permission for the job that it requires, not globally for all jobs.

We could add globally `packages: read` - that should be ok.